sdk/communication/Azure.Communication.JobRouter/tsp-location.yaml

```diff
@@ -1,4 +1,4 @@
-commit: 20e26266be77e6da4321259fb79e51e8281e48b2
+commit: 6a4cd586934a8478cc84c677afc49c4f149f8d87
```

Comment:
updated to a feature branch commit which only add `@convenientAPI(false)` to patch operations